.github/workflows/ci.yml

```diff
@@ -17,6 +17,9 @@ jobs:
       - name: Install NPM dependencies
         uses: bahmutov/npm-install@v1.7.4
 
+      - name: Build projects for eslint types
+        run: npm run build
+
```

Comment:
What's the purpose of this change?

Reply:
Eslint typecheck rules need the projects built to get cross-project types.